doc/nrf/drivers/wifi/nrf70_native.rst

```diff
@@ -118,3 +118,28 @@ To configure 15 dBm, use the value ``15``.
    };
 
 See the DTS binding documentation for more information.
+
+TX power calculation
+********************
+
```

Comment:
I suggest to add some intro text about transmit power before line 125, like: 
"This section describes how to determine the accurate transmit (TX) power on the nRF70 Series development platforms."

Reply:
Added suggested section intro text.